src/main/java/org/fundacionjala/at15/katas/pokerhands/jimy/PokerHandsDeck.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.at15.katas.pokerhands.jimy;
+
+public class PokerHandsDeck {
```

Comment:
As a first version of the solution, this looks good. From this point, you should start to consider thinking about new classes and methods that helps you to separate the logic. Don't forget to add relationships which rely on abstractions rather that concrete classes